extra/release_tool.py

```diff
@@ -3073,6 +3075,12 @@ def main():
         help="Find version(s) of integration repository that contain the given version of SERVICE, "
         + " where version is given with --version. Returned as a newline separated list",
     )
+    parser.add_argument(
+        "--feature-branches",
+        action="store_true",
+        default=False,
+        help="When used with -f, include upstream feature branches",
```

Comment:
suggested rewording: "-f" -> "-f or --integration-versions-including"

Reply:
I was trying to keep in in sync with https://github.com/mendersoftware/integration/blob/981ea7f696c245358c6d824659568d0855f4034c/extra/release_tool.py#L3126

I would then need to modify both, and the other one would get too long IMO.